client/src/main/scala/org/http4s/client/middleware/Metrics.scala

```diff
@@ -41,20 +44,28 @@ object Metrics {
       classifierF: Request[F] => Option[String])(
       req: Request[F])(implicit F: Sync[F], clock: Clock[F]): Resource[F, Response[F]] =
     (for {
+      statusRef <- Resource.liftF(Ref.of[F, Option[Status]](None))
       start <- Resource.liftF(clock.monotonic(TimeUnit.NANOSECONDS))
-      _ <- Resource.liftF(ops.increaseActiveRequests(classifierF(req)))
+      _ <- Resource.make(ops.increaseActiveRequests(classifierF(req)))(_ =>
+        ops.decreaseActiveRequests(classifierF(req)))
+      _ <- Resource.make(F.delay(())) { _ =>
```

Comment:
Can the first parameter just be an `F.unit`?

Reply:
Done! 😄